### ISSUE TITLE
define "high quality Image" on Image Component - low quality 

### ISSUE THREAD

**Issue by gitschwarz** [fttx/barcode-to-pc-app#314]
Actual: high quality image is defined as  an image with a height of 1080 and 80% jpeg quality ... That isn't really good and we can't print this image.
Can you add a simple define (serverside)  of high quality or expand the image component with the image resolution?
Thank you

**Comment by fttx** [comment]
Hello,

Yes, so you need a way to specify the target resolution, right?

**Comment by gitschwarz** [comment]
Yes - we make plant pictures for our webshop and for printing as labels. Webshop is ok, but the labelprint don't look good.
